### PR TITLE
Feature/textx2.0.1

### PR DESCRIPTION
**Pull request by goto40**
First adaptation to new registration API.
 * Part 01: from textx/tests
 * Part 02, 03: accordingly...

### CODE REVIEW

File: 01_separate_projects/flow_dsl/setup.py [hunk 1]
```diff
@@ -1,22 +1,12 @@
...
-      author_email='YOUR.NAME@ADDRESS',
-      license='TODO',
       packages=find_packages(),
       package_data={'': ['*.tx']},
-      install_requires=["textx", "arpeggio", "click", "types_dsl", "data_dsl"],
-      tests_require=[
-          'pytest',
-      ],
-      keywords="parser meta-language meta-model language DSL",
+      install_requires=["textx"],
```
Comment (igordejanovic) [inline 01_separate_projects/flow_dsl/setup.py]: Should `flow_dsl` require `data_dsl`?

File: 01_separate_projects/flow_codegen/tests/test_metamodel.py [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
-    assert(model is not None)
+    fname = os.path.join(this_folder,
+                         'models', 'data_flow.pu')
+    try:
+        os.remove(fname)
+    except OSError:
+        pass
+
+    model_file = os.path.join(this_folder,
+                              'models', 'data_flow.eflow1')
```
Comment (igordejanovic) [inline 01_separate_projects/flow_codegen/tests/test_metamodel.py:19]: Why do we need `1` suffix for all model files? It looks artificial.

Comment (igordejanovic) [reply 01_separate_projects/flow_codegen/tests/test_metamodel.py:19]: Oh, I get it now. You are enabling all three approaches to be installed/tested together. That makes sense.

File: README.md [hunk 3]
```diff
@@ -137,40 +137,11 @@ To install the metamodels and run the tests (on unix-like system), you can do th
 	py.test 02_shared_grammar/tests/                   # build
```
Comment (igordejanovic) [inline README.md:130]: There is still a section `Note on the CLI for the three metamodels` in the README that should be reworked or removed as it reference old dsl compiler.

Comment (igordejanovic) [reply README.md:130]: Also search for `types_data_flow_dslc`. There are some refrerences in `.pu` images. Those need update too.

**Review by igordejanovic — COMMENTED** [review]
Excellent! I have some comment/small issues I have found. Please see inline comments.

**Comment by igordejanovic** [comment]
BTW, I see you are doing some work on tkinter based model inspector in this repo. That sounds cool! ;)

File: 01_separate_projects/flow_dsl/setup.py (hunk 1 above)
Comment (goto40) [reply 01_separate_projects/flow_dsl/setup.py]: Yes. The same holds for data dsl. I will change this

**Comment by goto40** [comment]
I will also check the requirements* files.

File: 01_separate_projects/flow_dsl/setup.py (hunk 1 above)
Comment (goto40) [reply 01_separate_projects/flow_dsl/setup.py]: fixed

File: README.md (hunk 3 above)
Comment (goto40) [reply README.md:130]: fixed. I didn't think of them. Very good!

File: 01_separate_projects/flow_codegen/tests/test_metamodel.py (hunk 2 above)
Comment (goto40) [reply 01_separate_projects/flow_codegen/tests/test_metamodel.py:19]: (no action)

**Comment by goto40** [comment]
> BTW, I see you are doing some work on tkinter based model inspector in this repo. That sounds cool! ;)

Thanks ;-)
This is a separate experiment... Let's see if we can get something valuable from it.

**Comment by goto40** [comment]
Thank you for the review. If more issues are found, we can reopen a ne w ticket. Merging.

**Pull request merged**